spec/views/posts/index_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+RSpec.describe 'Post Index', type: :feature do
+  before :each do
+    @pic = 'https://whatsondisneyplus.com/wp-content/uploads/2022/09/andor-avatar.png'
+    @author = User.create(name: 'Momanyi Hassan',
+                          photo: @pic,
+                          bio: 'I’m a junior software developer! I can help you build a product, feature or website Look through some of my work and experience.')
+    @first_post = Post.create(author: @author, title: 'Selam from', text: 'This is my first post')
+    @second_post = Post.create(author: @author, title: 'Hello there!',
+                               text: 'This is ObiWans\' message: Don\'t let the Empire win')
+
+    @com1 = Comment.create(author: @author, post: @second_post, text: 'Thank you for your support 😊')
+    @com2 = Comment.create(author: @author, post: @second_post,
+                           text: 'Hello there Cassian, this Dero I am watching you')
+    @com3 = Comment.create(author: @author, post: @second_post, text: 'Spend the credits wisely :|')
+
+    visit user_path(@author)
+
+    click_on @author.name
+    click_link 'See all posts'
+  end
+
+  it 'Shows the username of the user' do
+    expect(page).to have_content(@author.name)
+  end
```

Comment:
* Kindly add Tests that assert you can see a section for pagination if there are more posts than fit on the view and 
* Tests that assert you can see the first comments on a post as these tests are missing

Reply:
resolved! ✔️